autogpt_platform/backend/backend/util/retry.py

```diff
@@ -20,7 +21,14 @@ def _log_prefix(resource_name: str, conn_id: str):
     return f"[PID-{os.getpid()}|THREAD-{threading.get_native_id()}|{get_service_name()}|{resource_name}-{conn_id}]"
 
 
-def conn_retry(resource_name: str, action_name: str, max_retry: int = 5):
+def conn_retry(
+    resource_name: str,
+    action_name: str,
+    max_retry: int = 5,
+    multiplier: int = 1,
```

Comment:
"the hell is a multiplier?" was my first reaction to reading this signature. `multiplier` -> `wait_multiplier`

Reply:
@ntindle,  @majdyz can respond when he's back online, but im gonna merge this to get the deploy out